envs/conda_build_config_p10.yaml

```diff
@@ -9,4 +9,4 @@ vector_settings:
 setuptools:
   - 63.*
 
-git_tag_for_env: open-ce-v1.8.0
+git_tag_for_env: open-ce-v1.8.1
```

Comment:
Let's remove this from conda_build_config files, although this won't cause any harm and I know it has been added since 1.8.0.